storage/src/journal/fixed.rs

```diff
@@ -91,7 +91,7 @@ pub struct Journal<E: Storage + Metrics, A: Array> {
     _array: PhantomData<A>,
 }
 
-impl<E: Storage + Metrics, A: Array> Journal<E, A> {
+impl<E: Storage + Metrics, A: Codec<Cfg = ()> + FixedSize> Journal<E, A> {
```

Comment:
Do we need to specify the default `Cfg=()`?

Reply:
Yeah it won't compile without that